storage/mysql/log_storage.go

```diff
@@ -18,32 +18,34 @@ import (
 
 const getTreePropertiesSQL string = "SELECT AllowsDuplicateLeaves FROM Trees WHERE TreeId=?"
 const getTreeParametersSQL string = "SELECT ReadOnlyRequests From TreeControl WHERE TreeID=?"
-const selectQueuedLeavesSQL string = `SELECT LeafHash,Payload
+const selectQueuedLeavesSQL string = `SELECT LeafRawHash,Payload
```

Comment:
or `TheDataHash`, just to make it clear that it's a hash of `TheData` (similar relative naming could happen in the `LeafData` struct.